### PR TITLE
typescript definitions (tsd)

### PR DESCRIPTION
**Pull request by ElemarJR**
type script definitions and some syntax test scenarios.

### CODE REVIEW

File: dist/poly2tri.d.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+﻿// Type definitions for poly2tri v0.9.10
```
Comment (r3mi) [inline dist/poly2tri.d.ts]: typo on the poly2tri version number, should be 1.3.5

**Comment by r3mi** [comment]
Thanks for the contribution, your interface is very clean !

Question regarding [DefinitelyTyped](https://github.com/borisyankov/DefinitelyTyped) : I see that the `poly2tri.d.ts` is also there ; what is the normal process to keep them in sync, when there is a change ? Should it be in only one place ? Is there an "established" way to manage this ?

**Comment by ElemarJR** [comment]
There is no "estabilished" way to make it sync. I think that every time .d.ts is updated here, we have to provide the update there.

**Comment by ElemarJR** [comment]
Sharing with you. I just used this Type Definition to create this (with BabylonJS)

screenshot

I created one "PR" to BabylonJS with this, that was accepted. BabylonJS 2 will use poly2tri to create triangulations

**Comment by r3mi** [comment]
looks nice, thanks for sharing

**Comment by r3mi** [comment]
thanks for this contribution

**Pull request merged**